tensorflow/compiler/xla/service/service.h

```diff
@@ -63,15 +63,16 @@ class ServiceOptions {
   int intra_op_parallelism_threads() const;
 
   // Sets the allowed_devices set for creation of stream executors.
```

Comment:
Same comment as in last review about comments that repeat verbatim what's in the code.  This occurs in a few places in this patch.

I'm not totally wild about repeating the comment explaining what allowed_devices means everywhere, but since it's only 3 lines or so, maybe that's better than having a pointer in the comments (i.e. it's better than "see Foo::set_allowed_devices()").

Reply:
`Same comment as in last review about comments that repeat verbatim what's in the code. This occurs in a few places in this patch.`, assuming you are talking about the space between getters-setters, done. If not please clarify what do you want me to do.

Reply:
Ok, I was confused by the comments above. I tried to improve it. I hope it is better, if not please suggest a better one and I will try to learn from that.